src/Nethermind/Nethermind.Trie/Pruning/TrieStore.cs

```diff
@@ -207,9 +209,10 @@ public Key(Hash256? address, in TreePath path, Hash256 keccak)
                     Keccak = keccak;
                 }
 
+                [SkipLocalsInit]
                 public override int GetHashCode()
                 {
-                    return HashCode.Combine(Keccak.GetHashCode(), Address?.GetHashCode() ?? 0, Path.GetHashCode());
+                    return (int)BitOperations.Crc32C((uint)Path.GetHashCode(), (uint)Keccak.GetHashCode()) + (Address?.GetHashCode() ?? 0);
```

Comment:
Why add `Address?.GetHashCode` and not mix it up?

Reply:
CRC seem to only have two parameter and the code I copy it from add the last one.